### DISCUSSION TITLE
RaspberryMatic Probleme nach HA OS Update

### DISCUSSION THREAD

**Discussion by teppic01** [OpenCCU/OpenCCU#1945] [🇩🇪 Fragen&Antworten] [answered]
Hallo Zusammen,
seit dem letzten Update für HA OS startet Raspberrymatic minutenlang. Es können auch keine HM Geräte mehr angelernt werden, nur noch HM-IP.

https://github.com/home-assistant/core/issues/78539#issue-1374519253

Downgrade zu 8.5 löst das Problem erst einmal.

Irgendwelche anderen Vorschläge?
Gruß

**Comment by jens-maus** [comment, marked as answer]
Siehe

https://github.com/jens-maus/RaspberryMatic/issues/1942

**Reply by teppic01** [reply]
Hi,
nachdem das Problem nach downgrade auf 8.5 gelöst war ist es seit dem letzten Update der Raspberrymatic wieder aufgetreten. Leider war ich im Urlaub und habe es nicht gemerkt. Schlecht für die Heizungssteuerung... :-(
Nach einigem hin und her testen lief es erst nach vollständigem update aus HASS 9 und dem aktuellen 22.10.1 nach mehrmaligem Neustart und einspielen eines Raspberrymatic Systembackup wieder. Vorher lief die Raspberrymatic nur mit ständigen Hängern. Außerdem hatte sie Benennung der Gruppen von Geräten "vergessen". Dadurch lief der Heizkessel durch...
Vielleicht hilft der Hinweis auf das update jemandem.
Gruß
T.

**Comment by TheJokerHA** [comment]
Hi,

bin mir echt nicht sicher wo oder wem ich das schreiben soll.
Wo der Fehler liegt kann ich leider auch nicht sagen ob HA der Fehler ist oder halt Raspberrymatic.
Dennoch denke ich hier könnte man nachforsche bzw. helfen!

HA.:
![Duty cycle sehr hoch 4](https://user-images.githubusercontent.com/85641621/196158116-6a036e2e-c35b-42b3-b0df-338640f095ee.JPG)

Duty cycle generiert die Verbindungsbrüche bzw. Raspberrymatic  hängt sich auf, warum auch immer!
![Duty cycle sehr hoch 2](https://user-images.githubusercontent.com/85641621/196159521-f6b1c63a-6582-40db-aaa0-253cb755f4ad.JPG)
![Duty cycle sehr hoch 3](https://user-images.githubusercontent.com/85641621/196159171-5b9e8fd8-a6ac-48c4-8aae-82effd7d41fe.JPG)
![Duty cycle sehr hoch](https://user-images.githubusercontent.com/85641621/196159332-57b55705-d923-4120-8d2c-899ba3acfa9e.JPG)

ab und zu mal sieht das auch so aus!
![Duty cycle sehr hoch 5](https://user-images.githubusercontent.com/85641621/196158744-ef8b5adb-0825-418f-9db6-2e2de934d21d.JPG)

aber es steigt stetig und erreicht dann die 100%, danach muss Raspberrymatic  neu gestartet werden.
